site/app/libraries/Utils.php

```diff
@@ -286,4 +286,33 @@ public static function removeStudentWithId($students, $key, $id) {
         }
         return $students;
    }
+
+   /**
+    * Convert the shorthand byte notation in php.ini to bytes.
+    * E.g : php return_bytes(ini_get('post_max_size'))
+    * Src : https://www.php.net/manual/en/function.ini-get.php
+    * @param string $val
+    * @return int 
+    */
+    public static function return_bytes ($size_str){
```

Comment:
Sorry that I failed to make comments all at once, but the function names should be in camelCase like other functions in this file per https://submitty.org/developer/coding_style_guide/php#naming-conventions